models/letter.ts

```diff
@@ -312,7 +312,7 @@ export class Letter {
    * @memberof Letter
    */
 
-  "return_envelope"?: string | boolean;
+  "return_envelope"?: string | boolean | null;
```

Comment:
pretty sure this is not a valid value for return envelope. Note that a property not being present is not the same as Null